src/query/ast/src/ast/statements/principal.rs

```diff
@@ -190,6 +190,8 @@ pub enum ShareGrantObjectName {
     Database(Identifier),
     // database name, table name
     Table(Identifier, Identifier),
+    // database name, table name
+    View(Identifier, Identifier),
```

Comment:
In GrantObject and OwnershipObject store table_id, db_id in meta. But in ShareGrantObjectName store identifier.

how to deal this case, If db/table/view rename?

Reply:
rename a share object will lose effect